botbot/apps/plugins/core/logger.py

```diff
@@ -10,6 +12,12 @@ class Config(config.BaseConfig):
         help_text="Don't log lines starting with this string"
     )
 
+def should_ignore_text(text, ignore_prefix):
+    return any(
+        (re.match(prefix, text, flags=re.IGNORECASE) is not None)
```

Comment:
interesting that you've gone for `re` instead of just doing `text.lower().beginswith(prefix.lower())`. To me it's a little easier to reason about, but I'm not sure which one is most efficient